microsoft-edge/toc.yml

```diff
@@ -1097,13 +1097,13 @@
             - name: Policies for Microsoft Edge Add-ons
               items:
                 - name: Content Security Policy (CSP)
-                  href: extensions-chromium/store-policies/csp.md
+                  href: /legal/microsoft-edge/extensions/csp
```

Comment:
Undo this TOC change (but see my other comment about renaming this article and moving it into the developer guide bucket.

Reply:
restored local path in toc; got removed from Files Changed